src/main/java/ifsul/agileproject/rachadinha/domain/entity/Racha.java

```diff
@@ -0,0 +1,59 @@
+package ifsul.agileproject.rachadinha.domain.entity;
+
+import jakarta.persistence.Entity;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Entity
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Data
+@Table(name = "Racha")
+public class Racha {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id_racha")
+    private Long id;
+
+    @NonNull
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "description", length = 255)
+    private String description;
+
+    @NonNull
+    @Column(name = "password")
+    private String password;
+
+    @NonNull
+    @Column(name = "goal")
+    private Double goal;
+
+    @Column(name = "balance")
+    private Double balance;
+
+    @Column(name = "members")
+    private ArrayList<User> members;

```

Comment:
Add the annotations about relation between the entities.

[Here](https://medium.com/thefreshwrites/manytoone-onetomany-mapping-in-jpa-32581d3c0f8a) a reference.

Reply:
Done.
https://imgur.com/a/JsUU3Me